freqtrade/tests/optimize/test_backtesting.py

```diff
@@ -49,77 +52,54 @@ def test_backtest_1min_ticker_interval(default_conf, mocker):
     assert not results.empty
 
 
-def trim_dataframe(df, num):
-    new = dict()
-    for pair, pair_data in df.items():
-        new[pair] = pair_data[-num:]  # last 50 rows
+def trim_dictlist(dl, num):
+    new = {}
+    for pair, pair_data in dl.items():
+        # Can't figure out why -num wont work
+        # new[pair] = pair_data[-num:]
```

Comment:
This doesn't work because `trim_dictlist()` gets called with -100, but you pass `-num` to the slice, so you end up with  `pair_data[100:]`.